packages/concerto-core/types/lib/introspect/mapvaluetype.d.ts

```diff
@@ -30,7 +31,7 @@ declare class MapValueType extends Decorated {
      * @private
      */
     private processType;
-    type: string;
```

Comment:
Looks like typescript is making an assumption here. Can we cast to String somewhere to improve this?